frontend/src/app/App.tsx

```diff
@@ -49,8 +49,16 @@ export const WHOAMI_QUERY = gql`
 
 const App = () => {
   const appInsights = getAppInsights();
-
   const dispatch = useDispatch();
+
+  // Check if the user is logged in, if not redirect to Okta
+  if (process.env.REACT_APP_OKTA_ENABLED === "true") {
```

Comment:
I think we need to explicitly check that Okta is enabled, because the demo and training envs don't require authentication. This flag seems like an appropriate use to me.

Reply:
Right! forgot about demo and training. I'll leave it in place then :+1: